### DISCUSSION TITLE
Welcome to sakura Discussions!

### DISCUSSION THREAD

**Discussion by berryzplus** [sakura-editor/sakura#1482] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by berryzplus** [comment]
これ、GitHubの新機能らしいです。
とりあえず、テンプレそのまま開始にしてみました。

**Comment by kasumikagari** [comment]
まさに [sakura/management-forum](https://github.com/sakura-editor/management-forum/) のような運用を想定されている機能のように見えます。

management-forumの方の存在についさっき気づいたのですが、有用な議論がされているのに誘導がなくて見つけづらいのがすごくもったいない気がしました。
GitHub Wikiのどこかからリンクするか、いっそこちらに移転してきてもらえるとうれしいです。
